pkg/artifact/registry.go

```diff
@@ -127,7 +127,15 @@ func (a registry) fetchRegistryData(launchableID launch.LaunchableID, version la
 	var registryResponse RegistryResponse
 	err = json.Unmarshal(respBytes, &registryResponse)
 	if err != nil {
-		return nil, auth.VerificationData{}, util.Errorf("Could not unmarshal JSON response from artifact registry: %s", err)
+		l := len(respBytes)
```

Comment:
I don't think I fully follow your proposal. Would you want to route all JSON (un)marshaling through a wrapper module that captures its input in its error messages?

I only did this to the API calls because those are the ones where the JSON document is truly ephemeral. And it's more likely that an HTTP response will be something else, like a text or HTML error message.

I suspect the complexity of creating a `p2json` module outweights the benefits of changing the error message.  If you're concerned about repeating the code to do `x[:Min(80, len(x))]`, that could be factored out, but I didn't think in this case it was worth creating a new module for it.